### PR TITLE
User settings

### PR DESCRIPTION
**Pull request by imobach**
Add settings persistence (local storage backed) to UserService.

### CODE REVIEW

File: src/app/index.js [hunk 1]
```diff
@@ -31,6 +32,10 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
     blockUIConfig.cssClass = 'block-ui block-ui-anim-fade consent-dialog';
     blockUIConfig.autoBlock = false;
   })
+  .config(['localStorageServiceProvider', function (localStorageServiceProvider) {
+    localStorageServiceProvider.setPrefix('jh');
```
Comment (jreidinger) [inline src/app/index.js]: in general looks good, I just do not get why prefix is jh?

Comment (ancorgs) [inline src/app/index.js]: janusHangouts. That's the namespace of the application (it was named that way in the beginning).

**Comment by jreidinger** [comment]
LGTM

**Comment by imobach** [comment]
Thanks! I've updated it again after discussing with @ancorgs to move setting user and room.

File: src/app/room/room.controller.js [hunk 2]
```diff
@@ -12,31 +12,31 @@
...
+    var params = {};
 
-      if (room === null || user === null) {
-        // Redirect to signin making sure room is included in the url
-        if (room !== null) {
-          params.room = room.id;
-        }
-        $state.go('signin', params);
+    console.log(room);
+    console.log(user);
```
Comment (ancorgs) [inline src/app/room/room.controller.js]: meec!

Comment (imobach) [inline src/app/room/room.controller.js]: Ops!

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**